data-pipeline/src/covid_variant_pipeline/assign_clades.py

```diff
@@ -1,30 +1,40 @@
+import datetime
 import json
+import os
 import subprocess
 from importlib import resources
 
+import polars as pl
```

Comment:
we could easily just do this in pandas, but I admit to being polars-curious, so I threw it in here as we're experimenting (https://pola.rs/)